docker/ci/determine_country.bash

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# (TODO:lhw) Put below things in usage and make this script more 'standalone'.
+#
+# This script is totally optional, it guesses whether we are in China.
+# It will do nothing if the test fails, otherwise it will export the
+# WE_ARE_IN_CHINA=true evn variable and do other things the user directed
+# in the script param.
+
+# Get the very small tool to figure out if we are in China
+# This package can't be installed by apt-get
+# c.f. https://unixmen.com/find-fastest-mirror-debian-derivatives/
+if ! which netselect >/dev/null; then
+    wget --progress=bar -O netselect.deb 'http://ftp.debian.org/debian/pool/main/n/netselect/netselect_0.3.ds1-28+b1_amd64.deb'
```

Comment:
It looks to me that we don't really need to install `netselect.deb` to compare networking speed.  Instead, I used ping

```
yi@WangYis-iMac:~ $ ping -c 4 mirrors.aliyun.com | tail -1| awk '{print $4}' | cut -d '/' -f 2
23.167
yi@WangYis-iMac:~ $ ping -c 4 mirrors.ustc.edu.cn | tail -1| awk '{print $4}' | cut -d '/' -f 2
368.682
yi@WangYis-iMac:~ $ ping -c 4 mirrors.archive.ubuntu.com | tail -1| awk '{print $4}' | cut -d '/' -f 2
148.939
```